devicetypes/smartthings/fibaro-heat-controller.src/fibaro-heat-controller.groovy

```diff
@@ -328,4 +334,9 @@ private getMaxHeatingSetpointTemperature() {
 
 private getMinHeatingSetpointTemperature() {
 	temperatureScale == 'C' ? 10 : 50
+}
+
+private changeTemperatureSensorStatus(status) {
+	def map = [name: "healthStatus", value: status]
```

Comment:
@greens this event doesn't mark device offline. What should I send instead?

Reply:
`name: "DeviceWatch-DeviceStatus", value: "offline"`

I think that should work. I don't know how it meshes with devices that are actively pinged, but give it a shot.